src/containers/Downloads/DownloadDetails.js

```diff
@@ -229,7 +230,7 @@ class ExperimentsView extends React.Component {
                       {metadataFields && metadataFields.length ? (
                         metadataFields.join(', ')
                       ) : (
-                        <i class="result__not-provided">
+                        <i className="result__not-provided">
```

Comment:
Good catch! I am surprised it was not reported by babel-loader.

Reply:
yes, I think I saw it with LGTM or Lint